elixir/apps/domain/lib/domain/flows.ex

```diff
@@ -250,11 +251,22 @@ defmodule Domain.Flows do
   end
 
   defp expire_flows(queryable) do
-    {_count, flows} =
-      queryable
-      |> Flow.Query.expire()
-      |> Repo.update_all([])
-
-    {:ok, flows}
+    {:ok, :ok} =
```

Comment:
This is the main change. We chunk results from the DB and broadcast them instead of updating them, causing a side effect to broadcast.

I chose to chunk here (which requires a transaction) just in case the number of flows to be expired is very large.

This is expected to go away very soon in the next broadcast refactor PR or two.